_episodes/035-daniel-stenberg.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Rust in cURL"
+date: 2021-09-03T23:00:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/rustacean-station-e035-daniel-stenberg.mp3
+duration: "55:25"
+length: "53290745"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+#  Rust in cURL  
+
+First time guest host, Allen Wyma talks with Daniel, the original author of cURL, about using Rust in cURL.
+
+- cURL is a command line tool and library for transferring data with URLs.
+- cURL and its data transfer core, libcurl is all written in C, which is known to be not memory safe. 
+- While it is almost impossibe to rewrite it into another language, offering a third-party library written in Rust could take a further step forward.
+
+## Contributing to Rustacean Station
+
+Rustacean Station is a community project; get in touch with us if you'd like to suggest an idea for an episode or offer your services as a host or audio editor!
+
+- Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+- Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+- Github: [@rustacean-station](https://github.com/rustacean-station/)
+- Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+## Highlights
+
+- Architecture of cURL & libcurl 
+- What brought Daniel to Rust? How long has he been using Rust?
+- What language was Daniel working in before that? 
+- Third party libraries written in Rust?
+- What benefits does Rust bring to cURL?
+
+## Resources 
+ - [Curl](https://curl.se/)
+ - [Daniel's Blog](https://daniel.haxx.se/blog/2021/08/09/nocais-apology/)
+ - [Project Gemini](https://gemini.circumlunar.space/)
```

Comment:
We generally include a Credits section at the bottom of the notes. See [here](https://raw.githubusercontent.com/rustacean-station/rustacean-station.org/master/_episodes/034-rust-1.52-1.53.md) for an example.